UnityProject/Assets/Scripts/Core/Editor/Inspector/MasterDeviceInspector.cs

```diff
@@ -27,13 +19,14 @@ public class MasterDeviceInspector : Editor
 			{ MultitoolConnectionType.LightSwitch, Color.yellow },
 		};
 
-		private void OnEnable()
+		public override void OnEnableInEditor(object target)
 		{
-			DeviceLinker.InitDeviceLists(((IMultitoolMasterable) target).ConType, forceRefresh: true);
+			var asd = (IMultitoolMasterable)target;
+			DeviceLinker.InitDeviceLists(asd.ConType, true);
 		}
 
 		[DrawGizmo(GizmoType.Selected | GizmoType.Active)]
-		private static void DrawGizmoConnection(IMultitoolMasterable device, GizmoType type)
+		public void DrawGizmoConnectionCALL(IMultitoolMasterable device, GizmoType type)
```

Comment:
This needs to be static for `[DrawGizmo]`.

https://docs.unity3d.com/ScriptReference/DrawGizmo.html